src/EA.Weee.Web/Areas/Producer/ViewModels/SellingTechniqueViewModel.cs

```diff
@@ -30,6 +30,21 @@ public SellingTechniqueType ToSellingTechniqueType()
             throw new InvalidOperationException();
         }
 
+        public string ToSellingTechniqueTypeString() 
```

Comment:
Can we not use Display() attribute on enum and use the GetDisplayName helper in cshtml?

Reply:
@stevelee-civica  @seanciv  It would be worth clarifying with James that if both is selected we need to display both options or "Both". When selecting it on the EE screen Both isn't an option that they don't select both its inferred by the fact you select both radio buttons.

Reply:
Confirmed with James
possible display texts
1: "Direct selling to end user (mail, order, internet etc)"
2: "Indirect selling (other)"
3: "Direct selling to end user (mail, order, internet etc) and Indirect selling (other)"

Reply:
@stevelee-civica  Ok as Sean mentions rather than hard coding the text would it be possible to use the display helpers to create that text. And need unit test for this method...

Reply:
refactored to using Html.DisplayNameFor